framework/ui/src/main/java/com/google/android/catalog/framework/ui/components/CardItem.kt

```diff
@@ -50,12 +51,20 @@ internal fun CardItem(
     tags: List<String> = emptyList(),
     owners: List<String> = emptyList(),
     minSDK: Int = 0,
+    selected: Boolean,
     onItemClick: () -> Unit
 ) {
     val enabled = Build.VERSION.SDK_INT >= minSDK
     ElevatedCard(
         modifier = modifier,
         enabled = enabled,
+        colors = if (selected) {
```

Comment:
Thanks for adding the `selected` distinction here! This should also change the semantics for accessibility, so that the card is "clickable" when showing just the list, and "selectable" when showing both the list and detail. (an example of doing something like that [here](https://github.com/android/user-interface-samples/blob/45459b3a6f03887618ea9f4387ab24054667e5f8/CanonicalLayouts/list-detail-compose/app/src/main/java/com/example/listdetailcompose/ui/ListDetailSample.kt#L191-L207))

Reply:
Thanks for the accessibility hint. I will add that